### DISCUSSION TITLE
vertical bar

### DISCUSSION THREAD

**Discussion by bobblkabb** [Alexays/Waybar#1969] [Q&A]
Hello,

I am looking for a vertical bar with proper text and icon alignment. Can't find any. 
Anybody got or know good setups?

BR,
Michael

**Comment by hakan-demirli** [comment]
I also couldn't find one so I made one.
https://github.com/hakan-demirli/dotfiles/tree/0e03d37ea4f1e07bc01f5d8b8d080da3c389c221/.config/waybar
![image](https://github.com/Alexays/Waybar/assets/78746991/4312ce5a-bd2f-4693-8dd9-891e0bf1e698)
